engine/installation/linux/debian.md

```diff
@@ -110,12 +112,23 @@ Docker from the repository.
     the last line.
     **Do not use these unstable repositories on production systems or for non-testing workloads.**
 
-    ```bash
-    $ sudo add-apt-repository \
-           "deb https://apt.dockerproject.org/repo/ \
-           debian-$(lsb_release -cs) \
-           main"
-    ```
+    - **Debian**:
+
+      ```bash
+      $ sudo add-apt-repository \
+             "deb https://apt.dockerproject.org/repo/ \
+             debian-$(lsb_release -cs) \
+             main"
+      ```
+
+    - **Raspbian**:
```

Comment:
related to the above comment, is there are reliable way to collapse these two since they're almost the same?